test/test_google/test_bigquery_copy.py

```diff
@@ -0,0 +1,116 @@
+# testing bigquery copy method of google class
```

Comment:
So I see why you put this in `test_google` but bigquery code is actually tested under `test_databases`. It's an unfortunate bit of organization since the bigquery *code* is not in the databases folder (I'm not entirely sure it makes sense to have a databases folder at all, but that's a decision that predates me.

Anyway, can you move this to the existing test file [here](https://github.com/move-coop/parsons/blob/main/test/test_databases/test_bigquery.py)? I'm not sure if there's anything in the existing test file you'd want to reuse to make your tests easier to read but otherwise these tests look fine. 

Apologies again for the confusing organization.